src/common/components/form/validationUtils.tsx

```diff
@@ -46,8 +49,30 @@ const validateDate = (value: string | number) => {
   }
 };
 
+/** validateyear()
+ * Validate input year. Dates in the future are not valid.
+ * @param {number} value Input value.
+ * @returns {string} translation key
```

Comment:
```suggestion
 * Validate input year. Dates in the future are not valid.
 * @param {number} value Input value.
 * @returns {string} translation key if validation error, otherwise empty string
```

Reply:
fixed